packages/frontend/src/components/page-not-found/PageNotFound.js

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import { Translate } from "react-localize-redux";
+import styled from "styled-components";
+
+import textBackgroundImage  from "../../images/bg-landing-patterned.svg";
+import FormButton from "../common/FormButton";
+import Container from "../common/styled/Container.css";
+
+const StyledContainer = styled(Container)`
+  &&& {
+    text-align: center;
+    padding-top: 20px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+
+    @media (max-width: 767px) {
+      margin-top: 35px;
+      padding: 35px 20px;
+      overflow: hidden;
+      margin-top: -13px;
+    }
```

Comment:
Looking great! ✌️ 

I think we can remove `L18 - L23` too as the main `Container` style should take care of all padding and margin.